doc/21-development.md

```diff
@@ -1736,9 +1736,11 @@ and don't care for the details,
 
 1. ensure there are 35 GB free space on C:
 2. run the following in an administrative Powershell:
-    1. `Enable-WindowsOptionalFeature -FeatureName "NetFx3" -Online`
+    1. `Enable-WindowsOptionalFeature -FeatureName NetFx3ServerFeatures -Online`
+       (Windows Server only)
```

Comment:
> Windows Server only

If this is something which only applies to some versions of an operating system, it should be written before the command as otherwise half the people would blindly copy the command just to be confused why it won't work on their non-server operating system.